scripts/update_submodules.sh

```diff
@@ -1,23 +1,24 @@
 #!/bin/bash
 
-# Fetch all submodule refs on a local machine. On the AWS Amplify build runner,
-# only execute a shallow fetch with a single branch.
-function fetch_submodules () {
-  if [[ -n ${AWS_APP_ID} ]]; then
-      git submodule update --init --remote --progress --depth 1 --single-branch
-  else
-      git submodule update --init --remote --progress
-  fi
-}
+# Make sure there is a subdirectory in content for each version named in
+# config.json
+DOCS_VERSIONS=$(jq -r '.versions[] | select(.deprecated != true) | .name' config.json);
+for v in $(echo "$DOCS_VERSIONS"); do
+   if [[ ! -d "content/$v" ]]; then
+       mkdir "content/$v";
+   fi
```

Comment:
`mkdir -p` will create dir conditionally

```suggestion
   mkdir -p "content/$v";
```